### PR TITLE
[GH-113] Implement delete feature in table view

### PR DESCRIPTION
**Pull request by arvinDarmawan**
#### Summary
When in table view there is no option to easily delete a task. Before you can delete a task you need to open it.

#### Ticket Link
Fixes https://github.com/mattermost/focalboard/issues/113

### CODE REVIEW

File: webapp/src/components/table/tableRow.tsx [hunk 1]
```diff
@@ -140,6 +147,28 @@ const TableRow = React.memo((props: Props) => {
...
+            <MenuWrapper
+                className='optionsMenu'
+                stopPropagationOnToggle={true}
+            >
+                <IconButton icon={<OptionsIcon/>}/>
+                <Menu position='left'>
+                    <Menu.Text
+                        icon={<DeleteIcon/>}
+                        id='delete'
+                        name={'Delete'}
```
Comment (kamre) [inline webapp/src/components/table/tableRow.tsx:160]: Should internalization be used for this `name`?

Comment (arvinDarmawan) [reply webapp/src/components/table/tableRow.tsx:160]: Hi @kamre , so I need to  add "TableRow.delete" in internalization file, right ?

Comment (kamre) [reply webapp/src/components/table/tableRow.tsx:160]: @arvinDarmawan yes, there are many other usages of `*.delete` as `id` for formated messages in other places.

So it looks like `TableRow.delete` is the correct name for `id` in this place.

**Comment by jespino** [comment]
@asaadmahmood can you take a look to the UX/Design of this?

File: webapp/src/components/table/tableRow.tsx (hunk 1 above)
Comment (arvinDarmawan) [reply webapp/src/components/table/tableRow.tsx:160]: Hi @kamre , I already check again in en.json file, and there are  "ContentBlock.Delete" and "TableHeaderMenu.delete".
Should we use delete with initial letter d or D ?

Comment (kamre) [reply webapp/src/components/table/tableRow.tsx:160]: @arvinDarmawan I don't have exact anwser, but it looks like most of other ids use lower case after the dot.

Comment (arvinDarmawan) [reply webapp/src/components/table/tableRow.tsx]: done @kamre , just waiting for UX/design

**Comment by asaadmahmood** [comment]
@jespino Added a comment on the main issue to point out how it should look like.
https://github.com/mattermost/focalboard/issues/113

**Review by hahmadia — APPROVED** [review]
From a code and usability perspective, I am approving because it's fine.

However, I'm not sure if UI changes will cause code changes. If they do, feel free to re-request a review from me. 

Thanks for the work either way 😄

**Comment by asaadmahmood** [comment]
@hahmadia It should look like this. So this may still need some work.
image

https://www.figma.com/file/nIyq6YPnKUxurCr0AyklZj/113-Delete-Tasks?node-id=101%3A7882


We can probably just move the button to the left with css, and position the dropdown to its right.

**Review by asaadmahmood — CHANGES_REQUESTED** [review]
Added a comment.

**Review by hahmadia — CHANGES_REQUESTED** [review]
Changing to Request Changes so I can be re-notified when the new UI changes have been made.

Thanks @asaadmahmood for heads up :)

**Comment by chenilim** [comment]
Trying to clean up stale PRs. @asaadmahmood, @hahmadia, are there still changes required? Or can they be made later? Thanks.

**Comment by sbishel** [comment]
no longer relevent.

**Pull request closed without merging**